app/src/org/commcare/dalvik/provider/CaseDataContentProvider.java

```diff
@@ -202,6 +208,48 @@ private Cursor queryCaseList(Uri uri, String[] projection, String selection, Str
         }
         return retCursor;
     }
+
+    private Cursor queryCaseAttachments(String caseId, String[] projection, String selection, String[] selectionArgs, String sortOrder) {
```

Comment:
The only argument used by this method's body is caseId. The other arguments are unused; please remove them.